src/assets/icons/CityBikeIcon.tsx

```diff
@@ -3,17 +3,24 @@ import React from 'react'
 function CityBikeIcon(): JSX.Element {
     return (
         <svg
-            width="24"
-            height="24"
-            viewBox="0 0 24 24"
-            fill="currentColor"
+            width="120"
+            height="120"
```

Comment:
Tenker width/height 1em for scaling med font size, mer consistent med resten av ikonene